tests/dialects/test_oracle.py

```diff
@@ -329,6 +329,34 @@ def test_hints(self):
         )
         self.validate_identity("INSERT /*+ APPEND */ INTO IAP_TBL (id, col1) VALUES (2, 'test2')")
         self.validate_identity("INSERT /*+ APPEND_VALUES */ INTO dest_table VALUES (i, 'Value')")
+        self.validate_identity(
+            "SELECT /*+ LEADING(departments employees) USE_NL(employees) */ * FROM employees JOIN departments ON employees.department_id = departments.department_id",
+            write_sql="""SELECT /*+ LEADING(departments employees)
```

Comment:
[Nit] we don't usually specify `write_sql`, but rather set the arg as a positional one.